### ISSUE TITLE
relation: 'and' is not working

### ISSUE THREAD

**Issue by kumarabhishek** [fastify/fastify-auth#54]
## 🐛 Bug Report

Npm module fastify-auth has issue and thus `relation: 'and'` is not working for multiple auth handler inside a route.

## To Reproduce

```js
preHandler: fastifyApp.auth([fastifyApp.verifyJWT, fastifyApp.verifyOauth], { relation: 'and' }),
```

## Expected behavior

Both verifyJWT & verifyOauth should get called and should be passed before a route is handled.

Paste the results here:

```js
Only first auth handler is called.
```

## Your Environment

- *node version*: 12
- *fastify version*: >=2.7.1
- *fastify-auth version*: >=0.5.0
- *os*: Mac

## Root cause
When npm module fastify-auth gets installed it has different code than what is in [src](https://github.com/fastify/fastify-auth/blob/master/fastify-auth.js). When i replaced node_modules/fastify-auth/fastify-auth.js, it started working

**Comment by Eomm** [comment]
Hi,
the first auth handler check is positive or negative?

**Comment by kumarabhishek** [comment]
@Eomm  First auth handler is positive.

**Comment by Eomm** [comment]
Could you provide a complete reproducible error?

**Comment by Eomm** [comment]
@kumarabhishek I have spotted the problem... we don't have released that feature yet!! It is only merged on master but not on npm.

I will publish it in few hours 😅

**Comment by kumarabhishek** [comment]
@Eomm Glad you got it. Can you publish to npm as patch or minor as per changes.

**Comment by Eomm** [comment]
Yes, but I found some nasty bugs on status code (check the PR), so when others team members will approve I can release it 👍🏼
I hope it will be today, I need that functionality too 😁

**Comment by Eomm** [comment]
Released in v0.6.0 👍
